bootstrap.sh

```diff
@@ -2,8 +2,7 @@
 
 echo Building the build tool...
 
-GO111MODULE=on GOBIN="$(pwd)/util" go get -u github.com/Harvey-OS/ninep/cmd/ufs
-# GO111MODULE=on GOBIN="$(pwd)/util" go get -f -u bldy.build/bldy
+(cd sys/src/go; GOBIN="$HARVEY/util" go install ./...)
```

Comment:
Run `go mod tidy` so that it'll remove `github.com/Harvey-OS/ninep` from top-level `go.mod` file?

Reply:
I'm glad you're here ... I don't understand modules, still :-)